shaders_js.go

```diff
@@ -0,0 +1,83 @@
+//+build netgo
+
+package engo
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+)
+
+type Shader interface {
+	Initialize(width, height float32)
+	Pre()
+	Draw(texture *js.Object, buffer *js.Object, x, y, rotation float32)
```

Comment:
You have to use "EngoEngine.gl.Texture" in the definition of the drawable / texture as well. In the comment I made on the top.